pinc/daily_page_limit.inc

```diff
@@ -0,0 +1,18 @@
+<?php
+include_once($relPath.'User.inc');
```

Comment:
Can you please add some doc-level comment about this file and perhaps leave some breadcrumbs about where one might poke to see other parts of the daily_page_limit functionality? (Also adding a "dpl == daily_page_limit" together would help if someone is grepping the code and trying to figure out WTH "dpl" is 😁

Reply:
Done!

Reply:
Great - thank you!